### PR TITLE
Add Swift Code Example

### PR DESCRIPTION
**Pull request by 1998code**
Added Swift / SwiftUI Example (for iOS) in Readme.
Thank you :)

### CODE REVIEW

**Comment by 1998code** [comment]
Screenshot 2024-07-05 at 8 10 16 PM

**Comment by 1998code** [comment]
@martenson Please help with merging this update in readme. Thank you.

File: README.md [hunk 1]
```diff
@@ -214,3 +214,37 @@ public static boolean isDisposable(InternetAddress contact) throws AddressExcept
...
+
+...
+
+checkBlockList(email: newValue) { isBlocked in
+    if isBlocked {
+        // Handle Blocked Event
+    } else {
+        // Handle Passed Event
+    }
+}
```
Comment (martenson) [inline README.md:249]: this seems superficial
```suggestion
```

**Comment by martenson** [comment]
@1998code Thanks! Please trim your code example and I'll be happy to merge

**Comment by 1998code** [comment]
@martenson Ok done :)

**Comment by martenson** [comment]
thanks @1998code !

**Pull request merged**